modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/AbstractPlannerTest.java

```diff
@@ -704,7 +682,7 @@ protected static TestTable createTable(String name, int size, IgniteDistribution
                 throw new IllegalArgumentException("'fields[" + i + "]' should be a class or a SqlTypeName");
 
             RelDataType type = fields[i + 1] instanceof Class ? TYPE_FACTORY.createJavaType((Class<?>)fields[i + 1]) :
-                TYPE_FACTORY.createSqlType((SqlTypeName)fields[i + 1]);
+                TYPE_FACTORY.createTypeWithNullability(TYPE_FACTORY.createSqlType((SqlTypeName)fields[i + 1]), true);
```

Comment:
Why changed? I see the test fails but can't realise why.

Reply:
Fixed